src/main/scala/beam/utils/google_routes_db/ReqRespMapping.scala

```diff
@@ -0,0 +1,210 @@
+package beam.utils.google_routes_db
+
+import akka.Done
+import akka.actor.ActorSystem
+import akka.http.scaladsl.common.{EntityStreamingSupport, JsonEntityStreamingSupport}
+import akka.stream.scaladsl._
+import beam.sim.BeamHelper
+import beam.sim.common.SimpleGeoUtils
+import beam.utils.csv.CsvWriter
+import beam.utils.google_routes_db.config.GoogleRoutesDBConfig
+import beam.utils.google_routes_db.{request ⇒ req, response ⇒ resp}
+import org.matsim.api.core.{v01 ⇒ matsim}
+
+import scala.collection.mutable.ArrayBuffer
+import scala.collection.{immutable, mutable}
+import scala.concurrent.duration._
+import scala.concurrent.{Await, ExecutionContext, Future}
+import scala.util.{Failure, Success}
+
+/**
+ * Output file: ReqRespMapping.csv
+ *
+ * Run with Gradle:
+ * ./gradlew :execute \
+ *   -PmainClass=beam.utils.google_routes_db.ReqRespMapping \
+ *   -PappArgs="['--config','src/main/scala/beam/utils/google_routes_db/config/google_routes_db.conf']" \
+ *   -PlogbackCfg=logback.xml
+ */
+object ReqRespMapping extends BeamHelper {
+
+  private implicit val system: ActorSystem = ActorSystem("google-routes-db")
+  private implicit val execCtx: ExecutionContext = system.dispatcher
+  private implicit val jsonStreamingSupport: JsonEntityStreamingSupport = EntityStreamingSupport.json()
+
+  private var csvWriter: CsvWriter = _
+
+  //noinspection DuplicatedCode
+  def main(args: Array[String]): Unit = try {
+    val (_, cfg) = prepareConfig(args, isConfigArgRequired = true)
+    val config = GoogleRoutesDBConfig(cfg)
+    val geoUtils = SimpleGeoUtils()
```

Comment:
In order to get a good precision there should be a way to provide CRS (localCRS) instead of using default in that class which is `epsg:26910`

Reply:
Added a config value